pytorch3d/csrc/pulsar/global.h

```diff
@@ -73,6 +73,7 @@ struct float2 {
 struct float3 {
   float x, y, z;
 };
+#ifndef WITH_CUDA
```

Comment:
I'm very confused. This condition looks to me not to do anything, because we are in the else block on line 60 from the ifdef on line 57.

So I would have thought this PR is harmless but also not helpful. If it fixes your build that's great, but an explanation would be useful. Is WITH_CUDA special somehow? Do the includes in line 58 and 59 not have matched conditional blocks?

Reply:
You're right. My bad. I should have checked `main` first before checking.. it seems like the latest stable release still has this problem but main doesn't